core/unit_test/TestViewMemoryAccessViolation.hpp

```diff
@@ -157,7 +157,7 @@ void test_view_memory_access_violations_from_device() {
   test_view_memory_access_violation(make_view<V6>(lbl), exec_space, prefix + ".*UNAVAILABLE");
   test_view_memory_access_violation(make_view<V7>(lbl), exec_space, prefix + ".*UNAVAILABLE");
   test_view_memory_access_violation(make_view<V8>(lbl), exec_space, prefix + ".*UNAVAILABLE");
-  auto* const ptr = reinterpret_cast<int*>(0xABADBABE);
+  int* const ptr = nullptr;
```

Comment:
```suggestion
  auto* const ptr = reinterpret_cast<int*>(0x0BADBABE);
```

Reply:
There does not appear to be any reason why one would prefer a cute pointer value here over just passing nullptr ...